spec/models/harvest_report_spec.rb

```diff
@@ -12,7 +12,13 @@
   let(:harvest_job)        { create(:harvest_job, harvest_definition:, pipeline_job:) }
 
   describe 'associations' do
-    it { is_expected.to belong_to(:pipeline_job) }
+    it { is_expected.to belong_to(:pipeline_job).optional }
+  end
+
+  describe '#initialize' do
+    it 'has the same name as its harvest definition' do
+      expect(subject.definition_name).to eq harvest_definition.name
```

Comment:
suggestion: [this](https://rspec.info/features/3-12/rspec-core/subject/one-liner-syntax/) could be of interest again here 😄